pkg/bootstrap/docker/up.go

```diff
@@ -45,6 +46,9 @@ const (
 	initialProjectDisplay = "My Project"
 	initialProjectDesc    = "Initial developer project"
 
+	defaultRedirectClient = "oauthclient/openshift-web-console"
```

Comment:
@fabianofranz 
Although I could use the const `OpenShiftWebConsoleClientID` defined in [pkg/cmd/server/origin/auth.go](https://github.com/openshift/origin/blob/master/pkg/cmd/server/origin/auth.go#L76), I did not want to import server packages, but WDYT?